### PR TITLE
updated readme - wrapper note

### PR DESCRIPTION
**Pull request by bidi47**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # dot-debugbar
 
+> [!IMPORTANT]
+> dot-debugbar is a wrapper on top of [maximebf/php-debugbar](https://github.com/maximebf/php-debugbar)
+>
+> ![OSS Lifecycle](https://img.shields.io/osslifecycle/maximebf/php-debugbar)
+
 DotKernel's debug bar built on top of [maximebf/php-debugbar](https://github.com/maximebf/php-debugbar).
```
Comment (alexmerlin) [inline README.md]:
```suggestion
## dot-debugbar badges
```

**Review by alexmerlin — CHANGES_REQUESTED** [review]

**Review by arhimede — CHANGES_REQUESTED** [review]
Please add in documentation the line with wrapper

**Review by arhimede — CHANGES_REQUESTED** [review]
Please add in documentation the line with wrapper
without the line
> [!IMPORTANT]

File: README.md (hunk 1 above)
Comment (bidi47) [reply README.md]: replaced with wrapper note

**Review by alexmerlin — APPROVED** [review]

**Review by arhimede — APPROVED** [review]

**Pull request merged**